addons/account/tests/test_chart_template.py

```diff
@@ -51,13 +80,23 @@ def create_tax_template(cls, name, template_name, amount):
         return cls._create_tax_template(template_name, name, amount, tag_name=None)
 
     @classmethod
-    def _create_tax_template(cls, tax_template_xmlid, name, amount, tag_name=None, chart_template_id=None):
+    def _create_tax_template(cls, tax_template_xmlid, name, amount, tag_name=None, chart_template_id=None, account_name=None):
         if tag_name:
             tag = cls.env['account.account.tag'].create({
                 'name': tag_name,
                 'applicability': 'taxes',
                 'country_id': cls.company.account_fiscal_country_id.id,
             })
+        if account_name:
+            account_vals = {
+                'name': account_name,
+                'code': 'TACT',
```

Comment:
You probably should reuse account_name here, so that the helper can be called multiple times without creating clashing account codes.